connman/src/network.c

```diff
@@ -78,6 +78,7 @@ struct connman_network {
 	bool connecting;
 	bool associating;
 	bool disconnecting;
+	bool require_all_ip_types;
```

Comment:
This could be renamed to be something better?

Reply:
Yeah, that was almost the same other option I had in mind for this. To deny or to allow.

Reply:
This is gone now. Resolving.